### ISSUE TITLE
make fails on Windows "No rule to make target"

### ISSUE THREAD

**Issue by ObiDriftKenobi** [Rubberduckycooly/Sonic-CD-11-Decompilation#95]
I can't be certain where the problem lies - if it's me, or if it's something that's expected. Compiling anything on Windows is pain.

Either way, today I tried compiling the project on my own to play with some fixes that aren't in /releases yet, but as it turns out, I'm dumb enough to not do that correctly. Sad times.

This is the output of make:
```
$ make CXXFLAGS=-O2 CXX=x86_64-w64-mingw32-g++ STATIC=1 -j5
make: pkg-config: No such file or directory
mkdir -p objects/dependencies/all/theoraplay
make: pkg-config: No such file or directory
mkdir -p objects/dependencies/all/upng
make: *** No rule to make target 'objects/SonicCDDecomp/Animation.cpp.o', needed by 'bin/soniccd'.  Stop.
make: *** Waiting for unfinished jobs....
x86_64-w64-mingw32-g++ -static -MMD -MP -MF objects/dependencies/all/theoraplay/theoraplay.c.d  -O2 -Idependencies/all/filesystem/include -Idependencies/all/theoraplay -Idependencies/all/upng dependencies/all/theoraplay/theoraplay.c -o objects/dependencies/all/theoraplay/theoraplay.c.o -c
x86_64-w64-mingw32-g++ -static -MMD -MP -MF objects/dependencies/all/upng/upng.cpp.d  -O2 -Idependencies/all/filesystem/include -Idependencies/all/theoraplay -Idependencies/all/upng dependencies/all/upng/upng.cpp -o objects/dependencies/all/upng/upng.cpp.o -c
```

Any retries after that only print this:
```
$ make CXXFLAGS=-O2 CXX=x86_64-w64-mingw32-g++ STATIC=1 -j5
make: *** No rule to make target 'objects/SonicCDDecomp/Animation.cpp.o', needed by 'bin/soniccd'.  Stop.
```

Why am I so dumb.

**Comment by vanfanel** [comment]
This happens on GNU/Linux, too:

also happens on GNU/Linux:

```
pi@raspberrypi:~/src/Sonic-CD-11-Decompilation $ CXXFLAGS="-march=armv8-a+crc+simd -mtune=cortex-a72" make -j4
mkdir -p objects/dependencies/all/theoraplay
g++ -MMD -MP -MF objects/dependencies/all/theoraplay/theoraplay.c.d -D_REENTRANT -I/usr/local/include/SDL2 -march=armv8-a+crc+simd -mtune=cortex-a72 -Idependencies/all/filesystem/include -Idependencies/all/theoraplay -Idependencies/all/upng dependencies/all/theoraplay/theoraplay.c -o objects/dependencies/all/theoraplay/theoraplay.c.o -c
mkdir -p objects/dependencies/all/upng
make: *** No rule to make target 'objects/SonicCDDecomp/Animation.cpp.o', needed by 'bin/soniccd'.  Stop.
make: *** Waiting for unfinished jobs....
g++ -MMD -MP -MF objects/dependencies/all/upng/upng.cpp.d -D_REENTRANT -I/usr/local/include/SDL2 -march=armv8-a+crc+simd -mtune=cortex-a72 -Idependencies/all/filesystem/include -Idependencies/all/theoraplay -Idependencies/all/upng dependencies/all/upng/upng.cpp -o objects/dependencies/all/upng/upng.cpp.o -c
```

**Comment by ObiDriftKenobi** [comment]
Appears to be fixed with commit 8e970fd
